src/elbo.jl

```diff
@@ -22,11 +22,12 @@ end
 MonteCarlo() = MonteCarlo(20)
 
 """
-    elbo(fx::FiniteGP, y::AbstractVector{<:Real}, fz::FiniteGP, q::AbstractMvNormal; n_data=length(y), method=Default())
+    elbo(svgp::SVGP, fx::FiniteGP, y::AbstractVector{<:Real}; num_data=length(y), method=Default())
```

Comment:
Side note - not necessarily to be resolved in this PR! I just noticed it here - but what would you think of the more explicit
```suggestion
    elbo(svgp::SVGP, fx::FiniteGP, y::AbstractVector{<:Real}; num_data=length(y), quadrature=DefaultQuadrature())
```
(and renaming `Quadrature` to `GaussHermite`)?

Reply:
It often defaults to `Analytic` though, so it's not necessarily quadrature

Reply:
Hm, I kinda thought of "quadrature" as integration in general. But how about `integration_method` then? is that too verbose?

Reply:
[Wikipedia:](https://en.wikipedia.org/wiki/Quadrature_(mathematics)
> In mathematics, quadrature is a historical term which means the process of determining area
:wink:

Reply:
Fair enough, didn't know that!

Reply:
Accidentally changed this in #32 instead :/

Reply:
no worries, could've been #39 as far as i'm concerned :joy: